src/core/less/variables/responsive.less

```diff
@@ -0,0 +1,30 @@
+// =================
```

Comment:
is this file still needed?

Reply:
Removed. Done.

Reply:
This one I'm not sure about?

Based on the rest of the changes I would say no, but based on the conversation of should the framework build a course that looks horrible and in this comments case unresponsive.